autogpt_platform/frontend/Dockerfile

```
@@ -19,7 +19,7 @@ RUN if [ -f .env ]; then \
     fi
 RUN pnpm run generate:api
 # In CI, we want NEXT_PUBLIC_PW_TEST=true during build so Next.js inlines it
-RUN if [ "$NEXT_PUBLIC_PW_TEST" = "true" ]; then NEXT_PUBLIC_PW_TEST=true pnpm build; else pnpm build; fi
+RUN if [ "$NEXT_PUBLIC_PW_TEST" = "true" ]; then NEXT_PUBLIC_PW_TEST=true NODE_OPTIONS="--max-old-space-size=4096" pnpm build; else NODE_OPTIONS="--max-old-space-size=4096" pnpm build; fi
```

I don't get the point of this condition, because if `if [ "$NEXT_PUBLIC_PW_TEST" = "true" ]` then `pnpm build` will see it in the environment, right? No need to explicitly prepend it to the command.
```suggestion
RUN NODE_OPTIONS="--max-old-space-size=4096" pnpm build
```